### ISSUE TITLE
Option to choose chain while creating publication

### ISSUE THREAD

**Issue by eccogrinder** [gnosis/tabula#258]
**Is your feature request related to a problem? Please describe.**
During the creation of a publication, it would be nice if users clearly knew which chain they are on before creation.

**Describe the solution you'd like**
Either have text in the UI that clearly indicates which chain you are creating your publication on, or have options in the UI to quickly switch chains (the latter of which would also clarify which chains are compatible with Tabula).

**Comment by auryn-macmillan** [comment]
I like this suggestion. There's certainly some nuance to it, but I suspect that making this an explicit choice during the publication setup is probably a good idea, rather than simply using the network exposed by the user's wallet.

**Comment by eccogrinder** [comment]
Yeah, I think this would help with both navigation (#265) and the ENS feature request I added (#276). If we add the chain info on the publication page, we could have folks add ENS info from the jump, which I think would be really appealing for folks.
